public/app/plugins/datasource/postgres/meta_query.test.ts

```diff
@@ -0,0 +1,26 @@
+import { describe, expect } from 'test/lib/common';
+import { PostgresMetaQuery } from './meta_query';
+import QueryModel from './postgres_query_model';
+
+describe('Postgres MetaQuery', () => {
+  describe('buildSchemaConstraint()', () => {
+    describe('when executing buildSchemaConstraint()', () => {
+      const postgresMetaQuery = new PostgresMetaQuery({ table: '', timeColumn: '' }, new QueryModel({}));
+      it('should return a schema restraint', () => {
+        expect(postgresMetaQuery.buildSchemaConstraint()).toEqual(
```

Comment:
I'm not quite sure about this test. I feel like that it doesn't really test anything. Maybe add a test like this.
`should wrap table_schema with quote_ident` or just add a comment to the buildSchemaConstraint function that explains why we added the quote_ident function.

Reply:
good call! yeah, after I wrote that test I realized it didn't really test much. Swapped it out for an inline comment 👍

Reply:
Awesome, thank you!